std/http/server.ts

```diff
@@ -180,24 +178,6 @@ export class Server implements AsyncIterable<ServerRequest> {
       await req.finalize();
     }
 
-    if (req === Deno.EOF) {
-      // The connection was gracefully closed.
-    } else if (err && req) {
-      // An error was thrown while parsing request headers.
-      try {
-        await writeResponse(req.w, {
-          status: 400,
-          body: encode(`${err.message}\r\n\r\n`)
-        });
-      } catch (_) {
-        // The connection is destroyed.
-        // Ignores the error.
-      }
-    } else if (this.closing) {
-      // There are more requests incoming but the server is closing.
-      // TODO(ry): send a back a HTTP 503 Service Unavailable status.
```

Comment:
Ditto, there is no request to send a 503 to unless it refers to a previous request whose `done` has already been awaited.